field_manager/fm_yaml.F90

```diff
@@ -31,493 +31,320 @@
 !> @{
 module fm_yaml_mod
 #ifdef use_yaml
+
 use yaml_parser_mod
+use mpp_mod, only: mpp_error, fatal
 implicit none
 private
 
-integer :: i, table_i, type_i, model_i, var_i, var_j, attr_j !< counters
-
-!> @}
```

Comment:
This line should be added back in, its a workaround for doxygen to be able to read in interfaces and add them to the module page.

Reply:
Added back in 02085437c8de6c6d895c878158f2186edf07eacd.